logger/logger.go

```diff
@@ -19,20 +19,44 @@ func SetLogger(l logr.Logger, name string) {
 }
 
 func Debugw(msg string, keysAndValues ...interface{}) {
-	defaultLogger.V(1).Info(msg, keysAndValues...)
+	WithLogger(defaultLogger).Debugw(msg, keysAndValues...)
 }
 
 func Infow(msg string, keysAndValues ...interface{}) {
-	defaultLogger.Info(msg, keysAndValues...)
+	WithLogger(defaultLogger).Infow(msg, keysAndValues...)
 }
 
 func Warnw(msg string, err error, keysAndValues ...interface{}) {
+	WithLogger(defaultLogger).Warnw(msg, err, keysAndValues...)
+}
+
+func Errorw(msg string, err error, keysAndValues ...interface{}) {
+	WithLogger(defaultLogger).Errorw(msg, err, keysAndValues...)
+}
+
+type withLogger struct {
```

Comment:
I think you can also do `type withLogger logr.Logger`, and then

`withLogger(defaultLogger).Errorw`